service/src/main/kotlin/fi/espoo/evaka/reports/ManualDuplicationReportController.kt

```diff
@@ -0,0 +1,108 @@
+// SPDX-FileCopyrightText: 2017-2023 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.reports
+
+import fi.espoo.evaka.Audit
+import fi.espoo.evaka.decision.DecisionType
+import fi.espoo.evaka.shared.ChildId
+import fi.espoo.evaka.shared.DaycareId
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.EvakaClock
+import fi.espoo.evaka.shared.security.AccessControl
+import fi.espoo.evaka.shared.security.Action
+import java.time.LocalDate
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ManualDuplicationReportController(private val accessControl: AccessControl) {
+    @GetMapping("/reports/manual-duplication")
+    fun getPlacementCountReport(
+        db: Database,
+        user: AuthenticatedUser,
+        clock: EvakaClock
+    ): List<ManualDuplicationReportRow> {
+        return db.connect { dbc ->
+                dbc.read {
+                    accessControl.requirePermissionFor(
+                        it,
+                        user,
+                        clock,
+                        Action.Global.READ_MANUAL_DUPLICATION_REPORT
+                    )
+                    it.setStatementTimeout(REPORT_STATEMENT_TIMEOUT)
+                    it.getManualDuplicationReportRows()
+                }
+            }
+            .also { Audit.ManualDuplicationReportRead.log() }
+    }
+
+    private fun Database.Read.getManualDuplicationReportRows(): List<ManualDuplicationReportRow> {
+        val sql =
+            """
+select supp_a.id                                       as supplementary_application_id,
```

Comment:
"Connected daycare" is the term that's widely used in the code base. It would be better to use that instead of "supplementary".